resolwe_bio/descriptors/sample2.yml

```diff
@@ -0,0 +1,387 @@
+- slug: sample2
+  name: Sample annotation
+  description: Sample meta data compliant with GEO and ENCODE databases.
+  version: 1.0.0
+  schema:
+    - name: sample_info
+      label: Sample information
+      group:
+        - name: description
+          label: Description
+          type: basic:string
+          required: false
+        - name: species
+          label: Species
+          type: basic:string
+          choices:
+            - label: Caenorhabditis elegans
+              value: Caenorhabditis elegans
+            - label: Cricetulus griseus
+              value: Cricetulus griseus
+            - label: Dictyostelium discoideum
+              value: Dictyostelium discoideum
+            - label: Drosophila melanogaster
+              value: Drosophila melanogaster
+            - label: Homo sapiens
+              value: Homo sapiens
+            - label: Mus musculus
+              value: Mus musculus
+            - label: Odocoileus virginianus texanus
+              value: Odocoileus virginianus texanus
+            - label: Rattus norvegicus
+              value: Rattus norvegicus
+            - label: Solanum tuberosum
+              value: Solanum tuberosum
+            - label: Other
+              value: other
+        - name: strain
+          label: Strain
+          type: basic:string
+          description: >
+            If you are using an animal model, this is strain information of that model, e.g. C57BL6,
+            Sprague Dawley, etc.
+          required: false
+          hidden: "sample_data.species != 'Mus musculus' && sample_data.species != 'Rattus norvegicus'"
+        - name: genotype
+          label: Genotype
+          type: basic:string
+          description: >
+            Genotype of your sample, e.g. P53+/-, Tg2576, etc.
+          required: false
+        - name: biosample_type
+          label: Biosample type
+          type: basic:string
+          required: false
+          description: >
+            This is where you put what your samples are derived from, e.g. cell line, tissue,
```

Comment:
This is where you put what your samples are derived.. Maybe it would be better "The source of samples/biological material, e.g. ..."